### PR TITLE
New Package: krakenx-0.0.3

### PR DESCRIPTION
**Pull request by Lolzen**

### CODE REVIEW

**Comment by cr6git** [comment]
````
$ colctl
Traceback (most recent call last):
  File "/usr/bin/colctl", line 7, in <module>
    import usb.core
ModuleNotFoundError: No module named 'usb'
````
Missing dependency.

File: srcpkgs/krakenx/template [hunk 1]
```diff
@@ -0,0 +1,16 @@
+# Template file for 'krakenx'
+pkgname=krakenx
+version=0.0.3
+revision=1
+noarch=yes
+build_style=python3-module
+hostmakedepends="python3-pip"
+makedepends="python3"
```
Comment (cr6git) [inline srcpkgs/krakenx/template]: I don't think `python3` is needed in the `makedepends`

Comment (Lolzen) [reply srcpkgs/krakenx/template]: i missed libusb, thanks for pointing that out.
On https://github.com/KsenijaS/krakenx libusb-32bit is specifically mentioned, is it necessary to declare 32 bit in the dependencies or better to omit it?

File: srcpkgs/krakenx/template [hunk 2]
```diff
@@ -0,0 +1,15 @@
+# Template file for 'krakenx'
+pkgname=krakenx
+version=0.0.3
+revision=1
+noarch=yes
+build_style=python3-module
+hostmakedepends="python3-pip"
+depends="python3 libusb"
```
Comment (cr6git) [inline srcpkgs/krakenx/template]: The problem is not `libusb` ...
What you need is: https://pypi.org/project/pyusb/

Comment (Lolzen) [reply srcpkgs/krakenx/template]: Thank you. Close and reject this for now, please.
I'm seemingly struggling to much.

Comment (cr6git) [reply srcpkgs/krakenx/template]:
````
# Template file for 'python3-pyusb'
pkgname=python3-pyusb
version=1.0.2
revision=1
noarch=yes
wrksrc="pyusb-${version}"
build_style=python3-module
pycompile_module="usb"
hostmakedepends="python3-setuptools"
depends="python3"
short_desc="Offers easy USB devices communication in Python"
maintainer="local <root@localhost>"
license="BSD-3-Clause"
homepage="https://pypi.org/project/pyusb/"
distfiles="${PYPI_SITE}/p/pyusb/pyusb-${version}.tar.gz"
checksum=4e9b72cc4a4205ca64fbf1f3fff39a335512166c151ad103e55c8223ac147362

post_install() {
	vlicense LICENSE
}
````
That worked for me.

**Pull request closed without merging**